src/Microsoft.InsightsGenerator/DataTransformation.cs

```diff
@@ -78,29 +78,29 @@ private string[] GetColumnLabels(DataArray array, out DataArray.DataType[] colum
             {
                 if (columnInfo.ContainsKey(DataArray.DataType.String))
                 {
-                    int minDistinctValue = -1;
-                    int minColumnIndex = -1;
-
+                    int maxDistinctValue = Int16.MaxValue;
```

Comment:
should this be Int32.MaxValue?  also, a common pattern for max is to set to the minimum possible value and then not have the extra check in the loop assuming all values will be greater.

Reply:
Yeah, I worked with Nara yesterday to change the logic of the slicer column select. The thought process behind this was that the categories(slicers) should have less distinct values than the input

Reply:
that makes sense.  thanks!